content/how-to-analyse-haskell-performance/index.md

```diff
@@ -0,0 +1,22 @@
++++
+title = "A Beginner's Guide To Analysing Haskell Application Performance"
+date = 2024-11-27
+[taxonomies]
+authors = ["Hécate"]
+categories = ["Cabal"]
+tags = ["Performance", "Profiling"]
++++
+
+The Cabal manual now has a guide on how to analyse the performance of Haskell applications.
```

Comment:
This is too ambitious, although it echoes the title of the guide. What happened, I think, is when reviewing the guide we managed to tone down the text but overlooked the title. The guide is more about **configuring a project for profiling** rather than "analysing performance". The latter is a vast topic (perhaps, the Haskell Optimization Book covers it in good detail), and I don't think it's even remotely covered in the guide.

Reply:
I agree with this comment. I think it would be fair to say that the guide clarifies which options to set where to get profiling enabled, but I wouldn't call it a guide to do profiling.

Reply:
Perfect, thank you for the feedback. I too was unsure about this title.